docs/services/data/_index.md

```diff
@@ -120,6 +120,31 @@ You can also control how local data is deleted if your machine's local storage b
 
 {{< /expand >}}
 
+{{< expand "Capture Directly to MongoDB" >}}
+
+Data Capture supports capturing tabular data directly to MongoDB in addition to capturing to disk.
+
+This feature is intended to support usecases like offline dashboards which don't require strong data
+delivery or consistency guarantees.
+
+When `mongo_capture_config.uri` is configured, Data Capture will attempt to connect to the configured MongoDB
+server and write captured tabular data to the configured `mongo_capture_config.database` and `mongo_capture_config.collection`
+(or their defaults if unconfigured) after enqueuing that data to be written to disk.
+
+If writes to MongoDB fail for any resson, Data Capture will log an error for each failed write and continue capturing.
+
+Failing to write to MongoDB doesn't affect capturing & syncing data to app.viam.com other than adding capture latency.
+
+{{< alert title="Caution" color="caution" >}}
+
+- Capturing Directly to MongoDB may write data to MongoDB that later fails to be written to disk (and therefore never gets synced to app.viam.com).
+- Capturing Directly to MongoDB does not retry failed writes to MongoDB. As a consequence, it is NOT guaranteed all data captured will be written to MongoDB. This can happen in cases such MongoDB being inaccessible to viam-server or writes timing out.
+- Capturing Directly to MongoDB may reduce the maximum frequency that Data Capture can capture data due to the added latency of writing to MongoDB. If your usecase needs to support very high capture rates, this feature may not be appropriate.
+
+{{< /alert >}}
```

Comment:
I'm confused why this is showing up in red with a title of `WARNING` rather than in yellow with a title of `Caution`:
 
![Screenshot 2024-11-13 at 12 40 09 PM](https://github.com/user-attachments/assets/75b45295-a250-4e59-89ae-18f034906964)

Reply:
It appears as if the demo site is not updating,

This is what is looks like locally when I run `make serve-dev`
![Screenshot 2024-11-13 at 1 16 57 PM](https://github.com/user-attachments/assets/972bd6f0-8d58-43f1-97e7-7d6dcd128785)

Reply:
it's now working now that I rebased & squashed